ledger/streaming/api/src/error.rs

```diff
@@ -20,6 +21,12 @@ pub enum Error {
 
     /// Invalid signature: {0}
     Signature(String),
+
+    /// DB Access Error
+    DBAccess(String),
```

Comment:
How about naming this `LedgerDb`? Either way, lowercase the `b` and include the string in the doc comment.
```suggestion
    /// Ledger DB: {0}
    LedgerDb(String),
```
```suggestion
    /// DB Access Error: {0}
    DbAccess(String),
```

Reply:
Good feedback, but I'll address this in a separate PR that's scoped for refactor. I'd like to keep PR changes in general in the domain of their subject matter. This came through because we had a ton of PRs in flight and there was some overlapping code in them (i.e. because both the validator and ledger sink used the ledger).